### DISCUSSION TITLE
Can't install the new version

### DISCUSSION THREAD

**Discussion by thebom999** [serenity-is/Serenity#6541] [Help] [answered]
I downloaded the serne file from vs market place. I have vs 2019, and when I try to run it. It says:
![image](https://user-images.githubusercontent.com/33629052/191099505-7a8c0dcd-ef3b-434c-8f21-670118c0a8a0.png)
PLEASE HELP

**Comment by dfaruque** [comment, marked as answer]
I think, latest serenity only supports visual studio 2022
